packages/fxa-settings/src/components/Settings/index.tsx

```diff
@@ -128,15 +128,16 @@ export const Settings = ({
   const { enabled: gleanEnabled } = GleanMetrics.useGlean();
 
   useEffect(() => {
-    !loading && gleanEnabled && GleanMetrics.pageLoad(location.pathname);
-  }, [loading, location.pathname, gleanEnabled]);
+    gleanEnabled && GleanMetrics.pageLoad(location.pathname);
+  }, [location.pathname, gleanEnabled]);
 
   if (loading) {
     return <LoadingSpinner fullScreen />;
   }
 
   // This error check includes a network error
   if (error) {
+    GleanMetrics.error.view({ event: { reason: error.message } });
```

Comment:
I went back and forth on where to put this view event (vs inside the component). I opted for here since there is still some work planned for the `AppErrorBoundary` and could see it changing from that.